akka-projection-grpc/src/main/scala/akka/projection/grpc/replication/javadsl/ReplicationSettings.scala

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.grpc.replication.javadsl
+
+import akka.actor.typed.ActorSystem
+import akka.annotation.ApiMayChange
+import akka.annotation.DoNotInherit
+import akka.annotation.InternalApi
+import akka.cluster.sharding.typed.ShardingEnvelope
+import akka.cluster.sharding.typed.javadsl.Entity
+import akka.cluster.sharding.typed.javadsl.EntityTypeKey
+import akka.grpc.GrpcClientSettings
+import akka.persistence.typed.ReplicaId
+import akka.projection.grpc.producer.EventProducerSettings
+import akka.projection.grpc.producer.javadsl.EventProducerInterceptor
+import akka.projection.grpc.replication.internal.ReplicaImpl
+import akka.projection.grpc.replication.scaladsl.{ ReplicationSettings => SReplicationSettings }
+import akka.util.JavaDurationConverters.JavaDurationOps
+import com.typesafe.config.Config
+
+import java.time.Duration
+import java.util.Optional
+import java.util.{ Set => JSet }
+import scala.jdk.CollectionConverters._
+import akka.projection.grpc.replication.internal.ReplicationProjectionProviderAdapter
+
+object ReplicationSettings {
```

Comment:
The reason we need concrete ReplicationSettings classes Java/Scala is the configure method which does a Entity => Entity transform to be able to configure the entity, and that is not possible to do with a Scala-Java back-and-forth transformation.